### PR TITLE
Added Swap Pool Balance and Contract Balance

### PR DESCRIPTION
**Pull request by chainupk**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/hashking exports TVL: 
        
 
 
    filecoin                  4.87 M
    
    total                    4.87 M

**Comment by llamatester** [comment]
The adapter at projects/hashking exports TVL: 
        
 
 
    filecoin                  4.88 M
    
    total                    4.88 M

File: projects/hashking/index.js [hunk 1]
```diff
@@ -7,7 +7,11 @@ module.exports = {
       const validators = await api.call({ target: owner, abi: 'function beneficiarys() public view returns (address [] memory)'})
       const bals = await api.multiCall({  abi: 'uint256:totalStakingFil', calls: validators})
       bals.forEach(i => api.add(nullAddress, i))
-      return sumTokens2({ api, owners: [owner], tokens: [nullAddress]})
+      const wFILContract = "0xD9A724840a46370c01a50C1E511087ab3a07FB53"
+      const totalSupply = await api.call({ target: wFILContract, abi: 'uint256:totalSupply'})
```
Comment (g1nt0ki) [inline projects/hashking/index.js:11]: why are you counting total wFIL supply as tvl?

Comment (chainupk) [reply projects/hashking/index.js:11]: wFIL would represent the amount in our liquidity pool.

Comment (g1nt0ki) [reply projects/hashking/index.js:11]: > wFIL would represent the amount in our liquidity pool.

isnt wFIL like WETH? why does that represent your liquidity pool, I am confused
image

**Pull request merged**